### ISSUE TITLE
iPhone X ScrollView Inset Issue

### ISSUE THREAD

**Issue by electic** [uias/Tabman#146]
Hi,

I am using TabMan and what I have is a UIViewController with a UIScrollView inside. The UIScrollview takes up the whole screen. When the iPhone X starts. The presenting UIView controller has the proper inset. However, if I were to swap tabs, the inset seems to be lost and the 50 to 60px of content disappear under the tab bar. Can anyone help?]

I am using XCode 9. Target Swift 3.2. TabMan 1.0.1.

I want to add this does not happen on an iPhone 8 or lower. Just seems to be related to the X.

-R

**Comment by electic** [comment]
So an update. I had to layout the uiscrollview and then explicitly call 

self.view.layoutIfNeeded()

then all tabs, not just the one initially shown, has the proper inset.

**Comment by mobru** [comment]
Similar here: I have some `UITableViewControllers`, but their contentInsets are off as soon as my `TabmanViewControllers` `viewDidAppear` is called (`top : 132.0, left : 0.0, bottom : 34.0, right : 0.0` on the iPhone X simulator).
It appears to be a problem with Tabman on iOS 11 in general (tested on iOS 11 simulator and device, as well as with an iOS 9 simulator where the insets are correct).

Also, `layoutIfNeeded()` did not fix the problem for me.

XCode 9, Swift 4.0, TabMan 1.0.1

**Comment by rnystrom** [comment]
Using this on my child VCs seems to fix the problem

```
if #available(iOS 11.0, *) {
  collectionView.contentInsetAdjustmentBehavior = .never
}
```

**Comment by hb852143** [comment]
[TabmanTableViewProblem.zip](https://github.com/uias/Tabman/files/1335558/TabmanTableViewProblem.zip)

Hi, I have same problem , I provide the sample project and screen shot.
Version: XCode 9, TabMan 1.0.1     Device: iphoneX Simulator and  iphone 8 Plus Simulator

![simulator screen shot - iphone 8 plus - 2017-09-27 at 10 00 52](https://user-images.githubusercontent.com/16014479/30892498-ac0f28c4-a36b-11e7-92b4-4d0d6173532f.png)
![simulator screen shot - iphone 8 plus - 2017-09-27 at 10 01 00](https://user-images.githubusercontent.com/16014479/30892496-ac0a018c-a36b-11e7-90b5-7382e0e5ec54.png)
![simulator screen shot - iphone x - 2017-09-27 at 10 00 08](https://user-images.githubusercontent.com/16014479/30892495-ac098c52-a36b-11e7-9b05-a88496ce2831.png)
![simulator screen shot - iphone x - 2017-09-27 at 10 00 27](https://user-images.githubusercontent.com/16014479/30892497-ac0befce-a36b-11e7-9dfe-c337c2eefc63.png)

**Comment by msaps** [comment]
Just released [1.0.2](https://github.com/uias/Tabman/releases/tag/1.0.2) which fixes the issues with insetting on iOS 11 and iPhone X! 😃

**Comment by electic** [comment]
Thank you so much!
